### PR TITLE
Sip 37 fix purgablesynth tests

### PR DESCRIPTION
**Pull request by hav-noms**

### CODE REVIEW

File: test/contracts/PurgeableSynth.js [hunk 1]
```diff
@@ -89,252 +91,231 @@ contract('PurgeableSynth', accounts => {
...
 
-							const expectedBalancePurged = await feePool.amountReceivedFromExchange(
-								effectiveValueOfPurgedSynths
-							);
-							assert.bnEqual(
-								userBalance,
-								expectedBalancePurged.add(usersUSDBalance),
-								'User must be credited back in sUSD from the purge'
-							);
+				describe('when the user exchanges 5000 of their sUSD into the purgeable synth', () => {
```
Comment (justinjmoses) [inline test/contracts/PurgeableSynth.js]: *when the user holds 5000 of the purgeable synth*

File: test/contracts/PurgeableSynth.js [hunk 2]
```diff
@@ -89,252 +91,231 @@ contract('PurgeableSynth', accounts => {
...
-								'The user must not be impacted by an empty purge'
-							);
-						});
-						it('and the totalSupply must be unchanged', async () => {
-							const iETHTotalSupply = await this.synth.totalSupply();
-							assert.bnEqual(
-								iETHTotalSupply,
-								totalSupplyBeforePurge,
-								'Total supply must be unchanged'
+							await exchangeRates.setInversePricing(
```
Comment (justinjmoses) [inline test/contracts/PurgeableSynth.js]: you could freeze it here instead of using a price update

File: test/contracts/PurgeableSynth.js [hunk 3]
```diff
@@ -89,252 +91,231 @@ contract('PurgeableSynth', accounts => {
...
+								iETH,
+								toUnit(100),
+								toUnit(150),
+								toUnit(50),
+								false,
+								false,
+								{ from: owner }
 							);
+							await exchangeRates.updateRates([iETH], ['160'].map(toUnit), timestamp, {
```
Comment (justinjmoses) [inline test/contracts/PurgeableSynth.js]: wait - is this freezing it? I don't see how it can

File: test/contracts/PurgeableSynth.js (hunk 2 above)
Comment (hav-noms) [reply test/contracts/PurgeableSynth.js]: I'll leave this as a natural freeze of via the system

**Pull request merged**